cron/cron-daily.pl

```diff
@@ -715,6 +715,7 @@ sub authors {
       require Encode;
       for (@$author) {
         defined && /\P{ASCII}/ && Encode::_utf8_on($_);
+        $_ //= ""; # some fields are undef, make them empty instead.
```

Comment:
Do all the warnings come from these lines below?

```perl
      s/\t/ /g for @$author[0,1,2];
      s/\t/%09/g for $author->[3];
```

That would imply undef userid, email, fullname, or homepage.

I looked at the database.  The only field ever null is `fullname`, and only five users have that.  I suspect it is no longer possible to achieve.  I think I'd rather we fix the data.  Then if it comes back, we know that we're allowing bogus data, but right now the oldest record like this is from 2008.

It's the work of 60s to fix the data in the database, updating five rows to set `fullname = ''`.  What do you think?